src/test/java/guitests/UITest.java

```diff
@@ -337,4 +337,11 @@ public GuiTest type(String text) {
         return this;
     }
 
+    /**
+     * Used to select the whole filter text so that it can be replaced
+     */
+    public void selectAll() {
+        pushKeys(KeyCode.END);
```

Comment:
But whether or not it's at the end, wouldn't Ctrl+A select everything? I was just wondering why it was needed.